code/modules/projectiles/bullet.dm

```diff
@@ -1891,21 +1891,17 @@ ABSTRACT_TYPE(/datum/projectile/bullet/homing/rocket)
 
 			T.hotspot_expose(700,125)
 			explosion_new(null, T, src.explosion_power, range_cutoff_fraction = 0.45)
-		return
-
-	is_valid_target(atom/A, obj/projectile/P)
-		. = ..()
-		return . && isliving(A) && !isintangible(A)
+			P.die()
```

Comment:
there a need to explicitly call `die()` here?

Reply:
I was finding rockets were hitting people laying down but still passing through them